### PR TITLE
Init

### PR DESCRIPTION
**Pull request by youngbogdanchik**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    "plugins": ["@typescript-eslint", "prettier", "react", "jest", "effector"],
+    "rules": {
+        "no-prototype-builtins": "off",
+        "eqeqeq": "error",
+        "react/prop-types": "off",
+        "radix": "warn",
+        "no-console": ["warn", { "allow": ["debug", "warn", "error", "assert"] }]
+    }
+}
```
Comment (mmospanenko) [inline .eslintrc.json:33]: use Prettier, "No new line..." error

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Logs
```
Comment (mmospanenko) [inline .gitignore:1]: keep only what you need here

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,8448 @@
+{
```
Comment (mmospanenko) [inline package-lock.json:1]: use yarn

File: babel.config.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
```
Comment (mmospanenko) [inline babel.config.js:1]: should be 2 spaces per tab

File: .yarnrc.yml [hunk 5]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
Comment (mmospanenko) [inline .yarnrc.yml:1]: not necessary

File: .prettierrc.json [hunk 6]
```diff
@@ -0,0 +1,8 @@
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "trailingComma": "all",
+  "singleQuote": true,
+  "printWidth": 100
+}
```
Comment (mmospanenko) [inline .prettierrc.json:8]: empty line

**Review by mmospanenko — APPROVED** [review]
init looks good, but fix some commented moments before merge

File: .eslintrc.json (hunk 1 above)
Comment (youngbogdanchik) [reply .eslintrc.json]: fixed

File: .gitignore (hunk 2 above)
Comment (youngbogdanchik) [reply .gitignore]: fixed

File: package-lock.json (hunk 3 above)
Comment (youngbogdanchik) [reply package-lock.json]: fixed

File: babel.config.js (hunk 4 above)
Comment (youngbogdanchik) [reply babel.config.js:1]: fixed

File: .yarnrc.yml (hunk 5 above)
Comment (youngbogdanchik) [reply .yarnrc.yml]: fixed

File: .prettierrc.json (hunk 6 above)
Comment (youngbogdanchik) [reply .prettierrc.json:8]: fixed

Comment (vladdoletskiy) [reply .prettierrc.json:8]: did not fix

**Review by vladdoletskiy — APPROVED** [review]

**Pull request merged**